### PR TITLE
Tecnologia

### PR DESCRIPTION
**Pull request by frangueriniwolox**
## Summary

- Added technology card component
- Added web assets
- Minor name refactor in colors
- Added font variables

## Screenshots

tech-card-component

## Trello card

https://trello.com/c/yNzoZBpK/22-tecnologia

### CODE REVIEW

**Comment by frangueriniwolox** [comment]
### Summary

- Fixed issues in the card margins by using flex-box instead
- Fixed position issues in the card

File: index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1> 
+  <h1 class="title">Front-End Tools</h1> 
+  <div class="tech-card">
+    <img class="tech-card-image" src="assets/react_icon.svg"></img>
```
Comment (SKOLZ) [inline index.html]: `img` tags are self closing: `` you can't put any children inside of it

File: index.html [hunk 2]
```diff
@@ -9,6 +9,12 @@
...
+    <h2 class="tech-card-title">React-Js</h2>
+    <p class="tech-card-release">Release: 2013</p>
```
Comment (SKOLZ) [inline index.html]: you could use a span here since it's a small piece of text and not an actual paragraph

File: scss/tech.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  font-family: $font-stack;
+  font-size: $text-medium;
+  position: relative;
+  text-align: center;
+}
+
+.tech-card:before {
+  border: 1px solid $earls-green;
+  border-radius: $tech-card-border-radius;
+  content: ' ';
```
Comment (SKOLZ) [inline scss/tech.scss]: you can remove the space between single quotes

File: scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  z-index: -1;
+}
+
+.tech-card {
+  align-items: center;
+  background: $earls-green;
+  border-radius: $tech-card-border-radius;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
```
Comment (SKOLZ) [inline scss/tech.scss]: this is not suported in most browsers. leave the default one(flex-start) instead. you want the space between elements to be generated by margins and not automatically

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  left: $tech-card-box-overlap;
+  min-height: $tech-card-box-min-height; 
+  padding: 0 20px;
```
Comment (SKOLZ) [inline scss/tech.scss]: I'd add some padding to the top and bottom

File: scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,53 @@
+%tech-card-item {
+  color: $mine-shaft;
+  font-family: $font-stack;
+  font-size: $text-medium;
+  position: relative;
```
Comment (SKOLZ) [inline scss/tech.scss]: this is not needed

File: scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,54 @@
+%tech-card-item {
+  color: $mine-shaft;
+  font-family: $font-stack;
+  font-size: $text-medium;
+  text-align: center;
+}
+
+.tech-card:before {
```
Comment (SKOLZ) [inline scss/tech.scss]: use ::before

File: scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  position: relative;
+  top: $tech-card-box-overlap;
+  width: $tech-card-box-width;
+}
+
+.tech-card-image {
+  @extend %tech-card-item; 
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+  margin: 20px 0;
```
Comment (SKOLZ) [inline scss/tech.scss]: you shouldn't need the margin top here

**Pull request merged**